bindings/experimental/distrdf/python/DistRDF/HeadNode.py

```diff
@@ -15,6 +15,7 @@
 from DistRDF.Node import Node
 from DistRDF.Operation import Action, InstantAction, Operation
 from DistRDF.Backends import Utils
+from DistRDF.Profiling import profilable_mapper
```

Comment:
We have to be wary about the "implicit" stuff happening to activate the profiling. You already took care of avoiding an implicit "import ROOT" when importing DistRDF (thanks!), but now I see that the `os.environ["CLING_PROFILE"]` variable is always set, at least in the local environment on the client side, due to this line. This line is running all the code in `DistRDF.Profiling.__init__`, which at line 19 sets the CLING_PROFILE variable. We need to understand whether that setting can be moved somewhere, hopefully inside the `profilable_mapper` would work?

Reply:
Unfortunately moving the statement doesn't seem to activate Cling's profiling feature. I added some comment in the latest commit